astacus/common/op.py

```diff
@@ -98,6 +98,8 @@ async def _async_wrapper():
                 op.set_status(Op.Status.running)
                 await fun()
                 op.set_status(Op.Status.done, from_state=Op.Status.running)
+            except ExpiredOperationException:
+                pass
```

Comment:
Maybe wrap the whole wrapper with `contextlib.suppress(ExpiredOperationException)`?

Reply:
Unfortunately that isn't possible, as it isn't async context manager (with just plain 'with ..', it would get ignored also in other async threads for duration of this async op).